### PR TITLE
Give a 'Team' column in numscale summary for instructors #2737

### LINKED ISSUE #2737: Give a 'Team' column in numscale summary for instructors
As per title, similar to #2672

### PR DESCRIPTION
**Pull request by rcyq**
Fixes #2737

### CODE REVIEW

**Comment by damithc** [comment]
@LowWeiLin  for final review please

**Comment by rcyq** [comment]
Changes made

**Comment by LowWeiLin** [comment]
Looks good.

File: src/main/webapp/deadlineExceededErrorPage.jsp [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <title>TEAMMATES</title>
     <link rel="stylesheet" href="/bootstrap/css/bootstrap.min.css" type="text/css"/>
     <link rel="stylesheet" href="/bootstrap/css/bootstrap-theme.min.css" type="text/css"/>
-    <link rel="stylesheet" href="stylesheets/teammatesCommon.css" type="text/css">
+    <link rel="stylesheet" href="/stylesheets/teammatesCommon.css" type="text/css">
```
Comment (damithc) [inline src/main/webapp/deadlineExceededErrorPage.jsp]: This change doesn't seem to be related to this issue.

Comment (rcyq) [inline src/main/webapp/deadlineExceededErrorPage.jsp]: So should I remove it and let #2869 fixes it?

Comment (damithc) [inline src/main/webapp/deadlineExceededErrorPage.jsp]: Yes, that's better. how did you come across this anyway? Just noticed the
missing slash accidentally?

Comment (rcyq) [inline src/main/webapp/deadlineExceededErrorPage.jsp]: It was during the refactoring. Somehow the piece of code that I was refactoring thrown NullPointerException back then. So when I took a look at the server side console, I discovered it.

Ok. I will leave this out from the patch.
